src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsPagesTest.java

```diff
@@ -990,10 +990,12 @@ else if (fieldClass == int[].class) {
                 }
 
                 if (isPropertyTokenType(sectionName, propertyName)) {
-                    result = "";
                     boolean first = true;
 
-                    if (value instanceof BitSet) {
+                    if (value == null) {
+                        result = "{}";
```

Comment:
Looking only at this commit, I am not seeing why this is needed as no docs were affected.
Alignment shouldn't have changed any properties.
Please explain.

Reply:
> Second commit to fix Default value is .

I am seeing it now. Please move changes for xdoc related to this change to this commit. This way the commits aren't connected together even though they are separate.

Reply:
I sort of question the `{}` since everything for these types are user friendly, written out. I am leaning to some type of display like `No tokens specified.` or some such.
What do you think @pbludov ?

Reply:
>What do you think @pbludov ?

It makes sense. I'll update the PR.

Reply:
@pbludov I am not seeing this item done.
> Please move changes for xdoc related to this change to this commit. This way the commits aren't connected together even though they are separate.

Reply:
done. Moved "no tokens" to second commit.